src/export.rs

```diff
@@ -65,20 +66,23 @@ pub fn write_html_out(image_file_path: &str, color_data: &[Color], out_file_path
 /// ```
 ///
 pub fn write_json_out(color_data: &[Color], out_file_path: &str) -> std::io::Result<()> {
-    let mut file = File::create(format!("{}.json", out_file_path))?;
-    file.write_all(b"{")?;
-    file.write_all(b"\"palette\": [")?;
+    let mut json: String = String::new();
+    json.push_str("{");
+    json.push_str("\"palette\": [");
 
     let mut color_it = color_data.iter().peekable();
 
     while let Some(color) = color_it.next() {
-        file.write_all(format!("{{ \"r\": {}, \"g\": {}, \"b\": {}, \"a\": {} }}", color.r, color.g, color.b, color.a).as_bytes())?;
+        json.push_str(format!("{{ \"r\": {}, \"g\": {}, \"b\": {}, \"a\": {} }}", color.r, color.g, color.b, color.a).as_str());
         if color_it.peek().is_some() {
-            file.write_all(b",")?;
+            json.push_str(",");
```

Comment:
Same `clippy` issue as in line 70.

```suggestion
            json.push(',');
```